packages/devcmd-cli/src/index.ts

```diff
@@ -54,9 +54,31 @@ async function isDir(path: string): Promise<boolean> {
 async function runInDevCmdsDir(dirPath: string) {
   const [, , ...args] = process.argv;
 
-  // TODO: use spawn or so instead
-  execFileSync("node", ["-e", `require('devcmd/from-cli').run(...process.argv.slice(1))`, ...args], {
-    cwd: dirPath,
-    stdio: "inherit",
+  try {
+    await startProcess('node', ["-e", `require('devcmd/from-cli').run(...process.argv.slice(1))`, ...args], dirPath);
+  } catch (err) {
+    process.exit(err);
```

Comment:
`err` might be an `Error` here (see line 74)

Reply:
Could you elaborate on what you think should happen when the caught err is an Error? 🤔 
![image](https://user-images.githubusercontent.com/25712873/125833563-9e73ea96-25aa-4b9b-8dca-992f9c6b50d7.png)

Obviously in the first case, something failed before executing the devcmd while in the second case the error occured during the execution of the devcmd itself causing the process to close with status code 1. But how should those cases be handled?

Reply:
My main concern was that you are using an `Error` object as a number in `process.exit`. 

This may be problematic, as the following example code has an status code 0:
```
try {
	throw new Error();
} catch(e) {
	process.exit(e);
}
```

I think I'd change line 74 to also throw/reject with a number (e.g. `1`), and maybe include some logging of the error.